request-signature-v2.go

```diff
@@ -243,13 +243,13 @@ var resourceList = []string{
 	"notification",
 	"partNumber",
 	"policy",
-	"response-content-type",
-	"response-content-language",
-	"response-expires",
+	"requestPayment",
 	"response-cache-control",
 	"response-content-disposition",
 	"response-content-encoding",
-	"requestPayment",
+	"response-content-language",
+	"response-content-type",
+	"response-expires",
 	"torrent",
 	"uploadId",
 	"uploads",
```

Comment:
Mention that this list is sorted already and mention that new fields added here should make sure to add at lexical location to this list. Otherwise can have signature related issues later.